.gitignore

```diff
@@ -14,7 +14,6 @@ __pycache__/
 .Python
 build/
 develop-eggs/
-dist/
```

Comment:
Please don't remove this folder for the sake of endpoint test. Compiled files should not be pushed to the repo